api/WebSocket.json

```
@@ -138,40 +120,40 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/web-sockets.html#dom-websocket-binarytype-dev",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "15"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "18"
             },
             "edge": {
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "11"
             },
             "firefox_android": {
-              "version_added": true
+              "version_added": "14"
```

```suggestion
              "version_added": "11"
```

Assuming there was no testing of older Firefox for Android, I think this should be 11 matching desktop. The other data is the same for desktop and Android, the only difference being in when the unprefixing happened. I kind of doubt that's correct, but that's a separate question.

---

There actually is no Firefox Android 11 through 13 release, which is why this is set to 14.

---

Oh :)